Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM golang:1.11-alpine3.7 as builder
+
+RUN apk add --no-cache --virtual .build-deps gcc musl-dev make git expat
+RUN apk add vips-dev --verbose --update-cache --repository https://dl-3.alpinelinux.org/alpine/edge/testing/ --repository https://dl-3.alpinelinux.org/alpine/edge/main
+
+ADD . /src
+WORKDIR /src
+RUN make dep
+RUN make test
+RUN make build
+
+#################
+FROM alpine:3.7
+ARG SERVER_ADDR=:8080
+ARG STORE_FILE_ORIGINALS=/images/originals
+ARG STORE_FILE_CACHE=/images/cache
+ARG STORE_FILE_THUMBNAILS=/images/thumbnails
+ARG STORE_S3_REGION=""
+ARG STORE_S3_BUCKET=""
```

Comment:
I don't think the ones concerning S3 are needed.

Reply:
OK